src/prefect_server/api/runs.py

```diff
@@ -287,9 +247,10 @@ async def get_or_create_task_run(
             states=[
                 models.TaskRunState(
                     tenant_id=task.tenant_id,
-                    **models.TaskRunState.fields_from_state(
-                        Pending(message="Task run created")
-                    ),
+                    state="Pending",
+                    timestamp=pendulum.now(),
+                    message="Task run created",
+                    serialized_state=Pending(message="Task run created").serialize(),
```

Comment:
Are we moving away from this pattern?

Reply:
(I see you've deleted it below -- alrighty)

Reply:
Yeah, it's an unnecessary function that hides what's actually happening.